### DISCUSSION TITLE
Add volta js tool manager

### DISCUSSION THREAD

**Discussion by wendevlin** [directus/directus#6910] [Feature Requests]
Hi,

I think [volta](https://volta.sh/) would improve the devolper experience.

You could pin in packages.json version of node, npm and lerna and developers just have to have volta installed and they get the current dev version. So all work on the same base.

**Comment by rijkvanzanten** [comment]
What problem would this fix? People develop using node in a number of different ways, ranging from a local install, a `n` or `nvm` managed environment, docker containers, remote execution online, etc etc, so hard-requiring a single tool to do that might not be worth the trade-off 🤔

**Reply by wendevlin** [reply]
Yeah good point that they are a lot of dev environments. But it can help devs to jump into development. 

Install volta, clone repo and you should have all the correct versions to develop. 

It's just some lines in the package json to let volta know your environment, if you don't want to use it everything works as before.

**Reply by rijkvanzanten** [reply]
Gotcha! Yeah we can add it to the root package.json for convenience sake 🙂 Don't want to enforce it though

**Comment by wendevlin** [comment]
I opened a PR: https://github.com/directus/directus/pull/6919

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
